util/libssl.num

```diff
@@ -403,3 +403,4 @@ SSL_dane_clear_flags                    403	1_1_0	EXIST::FUNCTION:
 SSL_SESSION_get0_cipher                 404	1_1_0	EXIST::FUNCTION:
 SSL_SESSION_get0_id_context             405	1_1_0	EXIST::FUNCTION:
 SSL_SESSION_set1_id                     406	1_1_0	EXIST::FUNCTION:
+DTLS_get_data_mtu                       407	1_1_1   EXIST::FUNCTION:DTLS
```

Comment:
Is this correct? I note no *other* functions appear to depend on DTLS; not even `DTLS_method()`...